src/commands/start-project.ts

```diff
@@ -143,6 +144,12 @@ const startProject = async (toolbox: RnBootstrapToolbox) => {
     }
   }
 
+  if (selectedOptions.icons === IconChoice.withIcons && 
+    (selectedOptions.styleLibrary === StyleLibraryChoice.StyleSheet || selectedOptions.styleLibrary === StyleLibraryChoice.StyledComponents)) {
+    print.highlight('Installing additional dependencies...');
+    await spawnProgress('yarn add react-native-svg@14.0.0');
```

Comment:
@AndyBuhchev based on lucide icon documentation we could use latest version 15.x.x